rosbag2_compression/test/rosbag2_compression/test_zstd_compressor.cpp

```diff
@@ -81,3 +83,28 @@ TEST_F(CompressionHelperFixture, zstd_compress_file_uri)
   EXPECT_GT(compressed_file_size, 0u);
   EXPECT_TRUE(rosbag2_storage::FilesystemHelper::file_exists(compressed_uri));
 }
+
+TEST_F(CompressionHelperFixture, zstd_decompress_file_uri)
```

Comment:
I don't see a test to check for bad input, i.e., the compressed file is invalid.

Reply:
Added unit test that checks for failure when a bad uri is passed.